src/platform/plugins/shared/discover/public/application/main/components/tabs_view/tabs_view.tsx

```diff
@@ -62,26 +85,76 @@ export const TabsView = (props: SingleTabViewProps) => {
     [items]
   );
 
+  if (hideTabsBar) {
+    return null;
+  }
+
+  void topNavMenuItems;
+
+  return (
+    <div
+      css={css`
+        width: 100%;
+        background-color: ${euiTheme.colors.lightestShade};
+      `}
+    >
+      <EuiFlexGroup gutterSize="xs" alignItems="center" wrap={false} responsive={false}>
+        <EuiFlexItem
+          grow={true}
+          css={css`
+            overflow: hidden;
+          `}
+        >
+          <UnifiedTabs
+            services={services}
+            items={items}
+            selectedItemId={currentTabId}
+            recentlyClosedItems={recentlyClosedItems}
+            unsavedItemIds={unsavedTabIds}
+            maxItemsCount={MAX_TABS_COUNT}
+            createItem={createItem}
+            getPreviewData={getPreviewData}
+            onChanged={onChanged}
+            onEBTEvent={onEvent}
+            onClearRecentlyClosed={onClearRecentlyClosed}
+          />
```

Comment:
I like this approach, it worked well when testing! In the final version, could we instead keep `UnifiedTabs` as a wrapper and pass `TopNavMenuBeta` in an `appendRight` prop or similar? That way it continues to apply the aria attributes, and we can ensure a consistent style for appended content in `UnifiedTabs`.